core/src/main/java/org/apache/brooklyn/core/typereg/BrooklynBomYamlCatalogBundleResolver.java

```diff
@@ -115,10 +115,16 @@ protected double scoreForNullFormat(Supplier<InputStream> f) {
 
         OsgiBundleInstallationResult result;
         try {
+
+            BasicManagedBundle basicManagedBundle = new BasicManagedBundle(vn.getSymbolicName(), vn.getVersionString(),
+                    null, BrooklynBomBundleCatalogBundleResolver.FORMAT,
+                    null, null);
+            // if the submitted blueprint contains tags, we set them on the bundle, so they can be picked up and used to tag the plan.
+            if( cm.containsKey("tags")) {
+                basicManagedBundle.tags().addTags((Iterable<?>)cm.get("tags"));
```

Comment:
Can `tags` value be a non-Iterable?

Reply:
Nope.

Reply:
@algairim  Tags should not be anything else but `Iterable`, but just in case somebody has made a mistake and put something else, we do not want to prevent the bundle from deploying. Tags are not really that important after all. So I added a condition for the tags to be added only when `Iterable`.

Reply:
Ok, convinced.